### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by ppetrosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,33 +180,32 @@ wakePin <- hardware.pinW;
 wakePin.configure(DIGITAL_IN_PULLDOWN, readBuffer);
 
 // Configure accelerometer
-accel.init();
 accel.setDataRate(100);
 
 // Configure the FIFO buffer in Stream Mode and set interrupt generator
 // to generate an interrupt when there are 30 entries in the buffer
-accel.configureFifoInterrupt(true, LIS3DH_FIFO_STREAM_MODE, 30);
+accel.configureFifoInterrupt(true, accel.FIFO_STREAM_MODE, 30);
```
Comment (betzrhodes) [inline README.md]: This doesn't look right - this should be a constant not a class variable?

File: README.md [hunk 2]
```diff
@@ -183,33 +180,32 @@ wakePin <- hardware.pinW;
...
-| --------- | ---- | ------------- | ----------- |
-| *enable*     | Boolean | N/A | `true` to enable, `false` to disable |
-| *threshold* | Float   | 2.0 | Inertial interrupts threshold in Gs |
-| *duration*  | Integer | 5 | Number of samples exceeding threshold<br>required to generate interrupt |
-| *options* | bitfield | *LIS3DH_X_HIGH* \| *LIS3DH_Y_HIGH* \| *LIS3DH_Z_HIGH* | See table below |
+| --- | --- | --- | --- |
+| *state* | Boolean | N/A | `true` to enable, `false` to disable |
+| *threshold* | Float  | 2.0 | Inertial interrupts threshold in Gs |
+| *duration* | Integer | 5 | Number of samples exceeding threshold<br>required to generate interrupt |
+| *options* | bitfield | *X_HIGH* \| *Y_HIGH* \| *Z_HIGH* | See table below |
```
Comment (betzrhodes) [inline README.md]: These should be namespaced - are these really the correct values?

File: README.md [hunk 3]
```diff
@@ -219,89 +215,89 @@ accel.configureInertialInterrupt(true, 1.0, 10, LIS3DH_X_LOW | LIS3DH_Y_LOW | LI
...
+### configureClickInterrupt(*state[, clickType][, threshold][, timeLimit][, latency][, window]*) ###
 
 This method configures the click interrupt generator.
 
 | Parameter | Type | Default Value | Description |
 | --- | --- | --- | --- |
-| *enable*    | Boolean | N/A | `true` to enable, `false` to disable |
-| *clickType* | Constant | *LIS3DH_SINGLE_CLICK* | *LIS3DH_SINGLE_CLICK* or *LIS3DH_DOUBLE_CLICK* |
+| *state* | Boolean | N/A | `true` to enable, `false` to disable |
+| *clickType* | Constant | *LIS3DH.SINGLE_CLICK* | *LIS3DH.SINGLE_CLICK* or *LIS3DH.DOUBLE_CLICK* |
```
Comment (betzrhodes) [inline README.md]: again these names look incorrect - aren't they namespaced constants now

File: README.md [hunk 4]
```diff
@@ -219,89 +215,89 @@ accel.configureInertialInterrupt(true, 1.0, 10, LIS3DH_X_LOW | LIS3DH_Y_LOW | LI
...
 | *threshold* | Float | 1.1 | Threshold that must be exceeded to be considered a click |
 | *timeLimit* | Float | 5 | Max time in milliseconds the acceleration can spend above the threshold to be considered a click |
-| *latency*   | Float | 10 | Min time in milliseconds between the end of one click event and the start of another to be considered a *LIS3DH_DOUBLE_CLICK* |
-| *window*    | Float | 50 | Max time in milliseconds between the start of one click event and end of another to be considered a *LIS3DH_DOUBLE_CLICK* |
+| *latency* | Float | 10 | Min time in milliseconds between the end of one click event and the start of another to be considered a *LIS3DH.DOUBLE_CLICK* |
```
Comment (betzrhodes) [inline README.md]: not the correct name

File: README.md [hunk 5]
```diff
@@ -219,89 +215,89 @@ accel.configureInertialInterrupt(true, 1.0, 10, LIS3DH_X_LOW | LIS3DH_Y_LOW | LI
...
+| *window* | Float | 50 | Max time in milliseconds between the start of one click event and end of another to be considered a *LIS3DH.DOUBLE_CLICK* |
```
Comment (betzrhodes) [inline README.md]: not the correct name

File: README.md [hunk 6]
```diff
@@ -219,89 +215,89 @@ accel.configureInertialInterrupt(true, 1.0, 10, LIS3DH_X_LOW | LIS3DH_Y_LOW | LI
...
 
-#### Single Click Example
+#### Single Click Example ####
 
 ```squirrel
 // Configure a single click interrupt
-accel.configureClickInterrupt(true, LIS3DH_SINGLE_CLICK);
+accel.configureClickInterrupt(true, LIS3DH.SINGLE_CLICK);
```
Comment (betzrhodes) [inline README.md]: not the correct name

File: README.md [hunk 7]
```diff
@@ -219,89 +215,89 @@ accel.configureInertialInterrupt(true, 1.0, 10, LIS3DH_X_LOW | LIS3DH_Y_LOW | LI
...
 ```
 
-#### Double Click Example
+#### Double Click Example ####
 
 ```squirrel
 // configure a double click interrupt
-accel.configureClickInterrupt(true, LIS3DH_DOUBLE_CLICK);
+accel.configureClickInterrupt(true, LIS3DH.DOUBLE_CLICK);
```
Comment (betzrhodes) [inline README.md]: not the correct name

File: README.md [hunk 8]
```diff
@@ -347,53 +343,54 @@ In the following example we setup an interrupt for free-fall detection:
...
 function sleep(timer) {
-    server.log("going to sleep for " + timer + " sec");
-    if (server.isconnected()) {
-        imp.onidle(function() { 
-           server.sleepfor(timer); 
-        });
-    } else {
-        imp.deepsleepfor(timer);
-    }
+  server.log("gGoing to sleep for " + timer + " sec");
```
Comment (betzrhodes) [inline README.md]: typo

**Review by betzrhodes — CHANGES_REQUESTED** [review]
The code has name-spaced constants that didn't get tracked into this doc update.

File: README.md (hunk 1 above)
Comment (ppetrosh) [reply README.md]: @smittytone can you please look into this one?

**Comment by ppetrosh** [comment]
@smittytone not all of @betzrhodes's questions have been addressed. So I'm leaving this open for now.

File: README.md (hunk 1 above)
Comment (smittytone) [reply README.md]: Sure

**Comment by smittytone** [comment]
Betsy's issues put right.

**Review by betzrhodes — APPROVED** [review]

**Pull request merged**